### DISCUSSION TITLE
Interrupted syncoid due to blk_update_request: I/O error - hostbyte=DID_NO_CONNECT driverbyte=DRIVER_OK

### DISCUSSION THREAD

**Discussion by slrslr** [jimsalterjrs/sanoid#793] [Q&A]
Hello,

on Debian 11 with 5.10.0-20-amd64
/usr/sbin/syncoid version 2.0.3

I ran:
syncoid -r --skip-parent --no-stream pool pool2
in the middle it frozen and "sudo journalctl -g sdd" shows the drive errors in title.

"sudo zpool status pool2 -v" shows SUSPENDED pool due to "IO failures"

"sudo zpool clear pool2" -> "cannot clear errors for pool2: I/O error"

Some people suggest additional clear parameters, but i do not want to break anything:
https://superuser.com/questions/685882/zpool-pool-i-o-is-currently-suspended/686823#686823

"lsblk" does NOT show the drive. Trying to reconnect it failed with [these journal messages](https://bin.disroot.org/?ad25bb70856c24ba#2U7EVU2ExkABQ2n2bg2xZJRDxqd413siwpnzkNKyW2pn) (mtp-probe device: 3 was not an MTP device).

After i have removed disk from my USB3 dock station and used other USB "dock" it loaded drive ($ lsblk) and "sudo zpool clear pool2" worked without error and sync automatically resumed.

Sync is too slow since USB 2.0 dock, can i safely interrupt "syncoid -r --skip-parent --no-stream pool pool2" ? Ctrl+C ?

Is there any lesson learned from this please or can i provide any particular detail?

Thank you

**Comment by jimsalterjrs** [comment]
This is not a syncoid or sanoid issue, this is a "help my hardware is malfunctioning" issue, which is way out of scope here. You would be better off asking for help in Reddit r/zfs (for ZFS specific help) or perhaps r/linux (for non-ZFS-specific help with your hardware), with a lot more eyeballs potentially on your request.

Note: I do **not** recommend using USB HDD docks for permanent access to drives. They're for *temporarily* hooking things up for access, not for 24/7 connection. USB connections are simply not stable enough for permanent, always-on storage.
